src/index.tsx

```diff
@@ -113,7 +114,7 @@ function Violation({
   return (
     <Popover ref={popoverRef} targetRef={targetRef} className="popover">
       <div className="controls">
-        <h1>Accessibility violation</h1>
+        <h2>Accessibility violation</h2>
```

Comment:
Just curious - is `h2` correct here? I don't see an `h1` anywhere on the page.

Reply:
On our example page, no, but I'd assume that a real site will have an `h1`. Since we won't know the context of our users' apps I just felt like `h2` was a better assumption (though we should also probably group the appended overlays into a section anyway and this becomes less of an issue).

Reply:
Ok. Makes sense, thanks!